packages/back-end/src/controllers/features.ts

```diff
@@ -762,12 +841,22 @@ export async function postFeatureDiscard(
   await discardRevision(revision, res.locals.eventAudit);
 
   const hasDrafts = await hasDraft(org.id, feature, [revision.version]);
+
   if (!hasDrafts) {
     await updateFeature(org, res.locals.eventAudit, feature, {
       hasDrafts: false,
     });
   }
 
+  const hasPendingReview = await pendingReview(org.id, feature, [
+    revision.version,
+  ]);
+  if (!hasPendingReview) {
+    await updateFeature(org, res.locals.eventAudit, feature, {
+      pendingReview: false,
+    });
```

Comment:
Instead of a 2nd `updateFeature` call, can we combine this with the `hasDrafts` logic above so it only has to hit Mongo once?